style.css

```diff
@@ -0,0 +1,184 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  margin: 0;
+}
+
+/* Header */
```

Comment:
- [ ] Kindly remove commented codes as commented codes make our code base not clean

Reply:
Actually, this was just for reference reasons, why would it mean something else?